babel.config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "presets": [
+    [
+      "@babel/env",
+      {
+        "useBuiltIns": "entry"
+      }
+    ],
+    [
+      "@babel/preset-react",
+      {
+        "pragma": "dom",
+        "pragmaFrag": "DomFrag",
```

Comment:
You don't work with the fragment, so maybe you don't need this option, By the way, I don't think that you need this options. You can try just `"runtime": "automatic"`. Also you can read more about options [here](https://babeljs.io/docs/en/babel-preset-react)